x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_integrations.tsx

```diff
@@ -122,31 +122,43 @@ export const AgentDetailsIntegration: React.FunctionComponent<{
         </EuiText>
       ),
       id: 'inputs',
-      children: packagePolicy.inputs
-        .filter((input) => input.enabled)
-        .map((input) => ({
-          label: (
-            <EuiToolTip
-              content={i18n.translate('xpack.fleet.agentDetailsIntegrations.viewLogsButton', {
-                defaultMessage: 'View logs',
-              })}
-            >
-              <StyledEuiLink
-                href={getHref('agent_details', {
-                  agentId: agent.id,
-                  tabId: 'logs',
-                  logQuery: getLogsQueryByInputType(input.type),
-                })}
-                aria-label={i18n.translate('xpack.fleet.agentDetailsIntegrations.viewLogsButton', {
-                  defaultMessage: 'View logs',
-                })}
-              >
-                {displayInputType(input.type)}
-              </StyledEuiLink>
-            </EuiToolTip>
-          ),
-          id: input.type,
-        })),
+      children: packagePolicy.inputs.reduce(
+        (acc: Array<{ label: JSX.Element; id: string }>, current) => {
```

Comment:
Nice 🔥 . No need to change but for TS you can also do `.reduce<Array<{ label: JSX.Element; id: string }>>(acc, current)`.